blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -75,6 +72,10 @@
     private TimeValue timeout;
     private static int MAX_CONTENT_LENGTH = 1048576; // 1 MB
 
+    private ServerBootstrap bootstrap;
+    private ChannelFactory channelFactory;
+    private ChannelGroup allOpenChannels = new DefaultChannelGroup("allOpenChannels");
```

Comment:
As far as I can see, the field ```bootstrap``` is never used. We just assign server to it. Do we need it? 
Also possibly we do not need ```channelFactory``` as an instance variable, do we? We need only ```allOpenChannels``` to be the instance variable since it's used to shutdown the server.